Core/Core/Extensions/SwiftUIExtensions.swift

```diff
@@ -38,7 +38,6 @@ public extension Image {
     }
 }
 
-@available(iOS, deprecated: 16.0, message: "Use accessibilityActions(_:) for managing accessibility actions instead.")
 public extension View {
     @ViewBuilder
     func `if`<Content: View>(_ condition: Bool, content: (Self) -> Content) -> some View {
```

Comment:
As I see this extension isn't used anywhere and is an anti-pattern so I suggest to remove it.

Reply:
It's used in `ListCellView L216 and L221`.

Reply:
Reworked.